src/backend/launcher.ts

```diff
@@ -269,6 +270,14 @@ async function prepareWineLaunch(
     }
   }
 
+  if (gameSettings.eacRuntime && !isInstalled('eac_runtime')) {
+    download('eac_runtime')
+  }
+
+  if (gameSettings.battlEyeRuntime && !isInstalled('battleye_runtime')) {
+    download('battleye_runtime')
```

Comment:
should we `await` these calls? so we don't launch the game before they are downloaded

also, should we handle the case when something fails to download? like no internet connection for example, since I understand that setting to true when it's not available is a problem

Reply:
agree on that as well.

Reply:
if you are offline then the anti cheat not running doesn't matter in most cases right?